tf-psa-crypto/core/psa_crypto_core.h

```diff
@@ -55,6 +55,15 @@ typedef enum {
     PSA_SLOT_PENDING_DELETION,
 } psa_key_slot_state_t;
 
+/* If the size of static key slots is not explicitly defined by the user, then
+ * set it to the maximum between PSA_EXPORT_KEY_PAIR_OR_PUBLIC_MAX_SIZE and
+ * PSA_CIPHER_MAX_KEY_LENGTH. */
+#if !defined(MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE)
+#define MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE  \
+    ((PSA_EXPORT_KEY_PAIR_OR_PUBLIC_MAX_SIZE > PSA_CIPHER_MAX_KEY_LENGTH) ? \
+     PSA_EXPORT_KEY_PAIR_OR_PUBLIC_MAX_SIZE : PSA_CIPHER_MAX_KEY_LENGTH)
+#endif /* !MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE*/
```

Comment:
To fix `check_doxygen_warnings`, I think that due to a limitation of how we build the documentation, you have to define `MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE` in a public header that is not `mbedtls_config.h`. That's how it's done for other non-boolean options, including `MBEDTLS_PSA_KEY_SLOT_COUNT`.

So please move the fallback definition from `psa_crypto_core.h` to `include/psa/crypto_extra.h`.